frontend/util.lua

```diff
@@ -1195,6 +1195,23 @@ function util.htmlEscape(text)
     })
 end
 
+function util.decodeSsid(text)
```

Comment:
It seems that this should either be a local function or a method over in NetworkMgr. util isn't anything that you could conceivably describe as a utility function but utility functions that have global appeal. ^_^

Reply:
Makes sense, I was on the fence about it anyway. I'll move it to NetworkMgr.